contribute-to-tidb/review-a-pr.md

```diff
@@ -7,46 +7,163 @@ This guide is for all contributors that want to help with reviewing code contrib
 * We make sure to keep a high code quality in TiDB.
 * We avoid situations where contributors and reviewers spend a lot of time refining a contribution that gets rejected later.
 
-Every review needs to check the following six aspects. We encourage to check these aspects in order, to avoid spending time on detailed code quality reviews when formal requirements are not met or there is no consensus in the community to accept the change.
 
-## 1. Is the Contribution Well-Described?
+## 1. Things to know before the code review
```

Comment:
Given that we don't use a checklist any more. It's better to remove the redundant `no.` prefix
```suggestion
## Things to know before the code review
```